Dotnet/AppApi/GameHandler.cs

```diff
@@ -64,6 +66,9 @@ public bool KillInstall()
             var processes = Process.GetProcessesByName("install");
             foreach (var p in processes)
             {
+                // Sometimes install.exe is suspended
+                ResumeProcess(p.Id);
```

Comment:
This would need to be in the `if (match.Success)` part of the code.
Again, to make sure we're not messing up any other processes by accident

Reply:
@lonelyicer 
```
// Sometimes install.exe is suspended
ResumeProcess(p.Id);
```

should be moved inside
```
if (match.Success)
```

so we don't affect other unrelated install processes